website/client/src/components/userMenu/profileModal.vue

```diff
@@ -35,15 +35,22 @@ export default {
       userId: undefined,
       startingPage: undefined,
       path: undefined,
+      selectedPage: '',
     };
   },
+  watch: {
+    startingPage () {
+      this.selectedPage = this.startingPage;
+    },
+  },
   mounted () {
     this.$root.$on('habitica:show-profile', data => {
       this.userId = data.userId;
       this.startingPage = data.startingPage || 'profile';
       this.path = data.path;
       this.$root.$emit('bv::show::modal', 'profile');
     });
+    this.selectPage(this.startingPage);
```

Comment:
why is this necessary?

Reply:
Actually I'm not seeing where this function is declare or `this.selectedPage` used

Reply:
Oh I must have left it by accident I was testing some code and forgot to take it out. Ill push a commit without that function.